package.json

```diff
@@ -37,22 +37,22 @@
   },
   "homepage": "https://github.com/amcdnl/ngxs#readme",
   "peerDependencies": {
-    "@angular/core": "^5.0.0",
-    "rxjs": "^5.5.6"
+    "@angular/core": ">=5.0.0 <7.0.0 || >6.0.0-beta <7.0.0",
+    "rxjs": ">=6.0.0 || ^5.6.0-forward-compat.0 || ^6.0.0-beta.1"
```

Comment:
Can we keep `5.5.6` here for current users or is that a hard req?

Reply:
yep it can be 5.5.6

Reply:
Can you update to include that then?

Reply:
oops , they have to at least go to 5.6.0-forward-compat.0 which is compatible with old libs